modules/apps/client-extension/client-extension-type-impl/src/main/java/com/liferay/client/extension/type/internal/factory/ThemeCSSCETImplFactoryImpl.java

```diff
@@ -63,21 +63,27 @@ public void validate(
 
 		if (!Validator.isBlank(baseURL) && !Validator.isUrl(baseURL, true)) {
 			throw new ClientExtensionEntryTypeSettingsException(
-				"please-enter-a-valid-base-url");
+				"Please enter a valid base URL",
+				"please-enter-a-valid-base-url"
```

Comment:
Same as above, all invalid url exceptions should follow the same pattern.

Reply:
For the base, clay, and main URL messages should they read like this:
				"Invalid URL: " + baseURL,
				"url-x-is-invalid", baseURL
Or should we edit the language key like this:
				"Invalid base URL: " + baseURL,
				"base-url-x-is-invalid", baseURL